src/scaffoldmaker/utils/bifurcation.py

```diff
@@ -3,15 +3,54 @@
 """
 
 from __future__ import division
-from opencmiss.utils.maths.vectorops import add, cross, mult, normalize, sub
+from opencmiss.utils.maths.vectorops import add, cross, dot, magnitude, mult, normalize, sub
 from opencmiss.zinc.element import Element, Elementbasis
 from opencmiss.zinc.field import Field
 from opencmiss.zinc.node import Node
 from scaffoldmaker.utils.eft_utils import remapEftNodeValueLabel, scaleEftNodeValueLabels, setEftScaleFactorIds
+from scaffoldmaker.utils.geometry import createCirclePoints
 from scaffoldmaker.utils.interpolation import DerivativeScalingMode, interpolateCubicHermite, interpolateCubicHermiteDerivative, \
-    smoothCubicHermiteDerivativesLine, interpolateLagrangeHermiteDerivative
+    interpolateCubicHermiteSecondDerivative, smoothCubicHermiteDerivativesLine, interpolateLagrangeHermiteDerivative
 
 
+def get_curve_circle_points(x1, xd1, x2, xd2, r1, rd1, r2, rd2, xi, dmag, side, elementsCountAround):
```

Comment:
create a curvature around the circle nodes (giving the points and the derivatives around the central point)